source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2185,14 +2189,15 @@ export abstract class Action {
 
     id: string;
     title: string;
+    isPrimaryAction: boolean;
 
     toJSON() {
         let result = {};
 
         Utils.setProperty(result, "type", this.getJsonTypeName());
         Utils.setProperty(result, "id", this.id);
         Utils.setProperty(result, "title", this.title);
-
+        Utils.setProperty(result, "isPrimaryAction", this.isPrimaryAction);
```

Comment:
Please remove. This is not an official Adaptive property so it can't just be emitted in toJSON

Reply:
This is outdated code. It is not from latest commit